winui/Rating/Placeholder-value.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Placeholder value in WinUI Rating control | Syncfusion
+description: Learn all about the placeholder feature in the Rating control to show the average rating value before the user provide their own rating.
+platform: WinUI
+control: SfRating
+documentation: ug
+---
+
+# Placeholder value in WinUI Rating
+
+The Rating control provides the average rating value before the user provide or click any other actions by using the `PlaceholderValue` property.
+
```

Comment:
@Priyanka-Vijayasankar By using the 'PlaceholderValue' property, the Rating control can display the average rating value before the user provides their own rating by click or other action.

Reply:
Updated the above changes.